WalletWasabi.Fluent/Behaviors/RestartLinuxRendererBehavior.cs

```diff
@@ -0,0 +1,31 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Controls;
+using Avalonia.Xaml.Interactivity;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+// TODO: This is a workaround for this Avalonia issue (https://github.com/AvaloniaUI/Avalonia/issues/11850)
+// Please, remove this Behavior and it's unique usage after it's been fixed.
+public class RestartRendererBehavior : DisposingBehavior<Window>
+{
+	protected override void OnAttached(CompositeDisposable disposables)
+	{
+		if (AssociatedObject is null)
+		{
+			return;
+		}
+
+		this.WhenAnyValue(x => x.AssociatedObject!.WindowState)
+			.Where(state => state == WindowState.Normal)
+			.Where(_ => OperatingSystem.IsLinux())
+			.Do(_ =>
+			{
+				AssociatedObject.Renderer.Start();
+				AssociatedObject.Activate();
```

Comment:
Why calling `Activate()` needed?

Reply:
Activate() is needed because in Linux, setting the `WindowState` behaves differently and doesn't bring the window to the front, making it difficult to see that something happened. This different behavior can be part of the bug I filed in Avalonia's repo.